tests/queries/0_stateless/02808_custom_disk_with_user_defined_name.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Tags: no-fasttest
+# Tags: no-fasttest, no-upgrade-check
```

Comment:
AFAIK it will not help because Upgrade check takes tests from the old branch, `no-upgrade-check` tags have no effect after https://github.com/ClickHouse/ClickHouse/pull/44879